### DISCUSSION TITLE
IBM PC XT with PC DOS 2.00

### DISCUSSION THREAD

**Discussion by Pikarko** [86Box/86Box#4000] [Q&A] [answered]
In reality, the PC XT has a built-in 10 MB hard drive, and I want to mount the same hard drive and run PC DOS 2.00.  But a problem arose, DOS simply does not see the hd (I get the “Invalid drive specification” error), I used different controllers, and in the BIOS I often saw the presence of a hard drive, but I still could not interact with it in the system.  Help me figure out this problem, because I haven’t found anything about this on the Internet.

**Comment by TC1995** [comment, marked as answer]
Have you tried the Xebec one (IBM PC)?

**Reply by Pikarko** [reply]
I think I tried all, but maybe I did something wrong, so I will try. Thanks for the tip

**Reply by TC1995** [reply]
Have you set the HDD bus to MFM/RLL?

**Reply by Pikarko** [reply]
> Have you set the HDD bus to MFM/RLL?

Yes

**Reply by TC1995** [reply]
Is it 0:0? Is the CHS of the 10MB disk 306/4/17?

**Reply by Pikarko** [reply]
> Is it 0:0? Is the CHS of the 10MB disk 306/4/17?

Yes it is

**Reply by Fenix770** [reply]
Dumb question:
Did you use the DOS DEBUG command utiltity?
Here are the instructions: https://minuszerodegrees.net/ibm_xebec/ibm_xebec_llf_debug.htm

Then you should use FDISK to create the partition and FORMAT.

**Reply by Pikarko** [reply]
> Dumb question:
> Did you use the DOS DEBUG command utiltity?
> Here are the instructions: https://minuszerodegrees.net/ibm_xebec/ibm_xebec_llf_debug.htm
> 
> Then you should use FDISK to create the partition and FORMAT.

Thank you, I'm new to this program and just started learning about DOS, so now it’s quite difficult to understand such a large number of details
